### ISSUE TITLE
 Exception while invoking method '/cards/remove' ReferenceError: Subtasks is not defined

### ISSUE THREAD

**Issue by okobloko** [wekan/wekan#1986]
## Issue

Wekan docker container: quay.io/wekan/wekan

Edited an issue and tried to move it... then it crashed.  I'm afraid that I can't be more precise than that - I wasn't expecting it!

Any advice on how to rescue things?!?

Exception while invoking method '/cards/remove' ReferenceError: Subtasks is not defined
    at cardRemover (models/cards.js:1007:3)
    at Object.Cards.after.remove (models/cards.js:1056:5)
    at packages/matb33_collection-hooks.js:574:20
    at Array.forEach (<anonymous>)
    at Function._.each._.forEach (packages/underscore.js:139:11)
    at packages/matb33_collection-hooks.js:573:11
    at Array.forEach (<anonymous>)
    at Function._.each._.forEach (packages/underscore.js:139:11)
    at after (packages/matb33_collection-hooks.js:572:9)
    at Object.<anonymous> (packages/matb33_collection-hooks.js:588:5)
    at Object.collection.(anonymous function) [as remove] (packages/matb33_collection-hooks.js:146:21)
    at ns.Collection.CollectionPrototype._validatedRemove (packages/allow-deny/allow-deny.js:406:34)
    at DDPCommon.MethodInvocation.m.(anonymous function) (packages/allow-deny/allow-deny.js:173:46)
    at packages/check.js:128:16
    at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1186:15)
    at Object._failIfArgumentsAreNotAllChecked (packages/check.js:127:41)

**Comment by xet7** [comment]
@okobloko 

Try:
1. Stop and Start Docker
2. Export board to Wekan JSON
3. Import Wekan JSON

Does it work?

**Comment by okobloko** [comment]
Unfortunately "the site cannot be reached" after stopping/restarting the container.  Something in mongo-db is borked. I do have backups in which I export the board using the WeKan API - but before I try to reimport that, I'm going to try to overwrite the mongodb _data dir with my last backup.

**Comment by xet7** [comment]
@okobloko 

Some Wekan users avoid using Subtasks and Custom Fields because they still have some bugs.

For example with subtasks, this does break board:
1. Create subtask on Wekan board.
2. On subtask board, move card to other board.
3. Because Wekan board can't find subtask anymore, Wekan board gets broken.

**Comment by okobloko** [comment]
I'm not actually using Subtasks or Custom Fields. I suppose I should have mentioned that ! :-)
Perhaps it was because I am using Swimlanes?  I was only pulling the task from one row to another in the same column though...

**Comment by xet7** [comment]
@okobloko 

There are some Swimlane bugs like #1971 #1887 #1784 but I don't think those would break Wekan board. If you find out how you did get Wekan board to break, please add info as comment to this issue.

**Comment by okobloko** [comment]
I will try to figure that out, if I can resurrect the board.

**Comment by xet7** [comment]
@okobloko 

I would guess that your old backup of Wekan would work if you remove all Subtask related data from MongoDB.

**Comment by okobloko** [comment]
So, it seems like this was caused by a chain of unconnected events.  It turns out that the crappy domain hoster I use just wiped out all my DNS settings.  I suspect that this happened while I was logged into WeKan.  It could even be that this happened between the (multiple?) calls required to actually move the task (it looks like it's quite AJAX heavy?).  So sorry for wasting your time.  I've restored the DNS settings, and it is working again.
